docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,628 @@ license: |
   limitations under the License.
 ---
 
-Aggregate functions
\ No newline at end of file
+Spark SQL provides build-in aggregate functions defined in the dataset API and SQL interface. Aggregate functions
+operate on a group of rows and return a single value.
+
+Spark SQL aggregate functions are grouped as <code>agg_funcs</code> in Spark SQL. Below is the list of functions.
+
+**Note:** All functions below have another signature which takes String as a column name instead of Column.
+
+* Table of contents
+{:toc}
+<table class="table">
+  <thead>
+    <tr><th style="width:25%">Function</th><th>Parameters</th><th>Description</th></tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td> <b>{any | some | bool_or}</b>(<i>c: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns true if at least one value is true</td>
+    </tr>
+    <tr>
+      <td> <b>approx_count_distinct</b>(<i>c: Column[, relativeSD: Double]]</i>)</td>
+      <td>Column name; relativeSD: the maximum estimation error allowed.</td>
+      <td>Returns the estimated cardinality by HyperLogLog++</td>
+    </tr>   
+    <tr>
+      <td> <b>{avg | mean}</b>(<i>c: Column</i>)</td>
+      <td>Column name</td>
+      <td> Returns the average of values in the input column.</td> 
```

Comment:
super nit: could you remove unnecessary spaces? e.g., `<td> Returns...`

Reply:
sure